README.md

```diff
@@ -102,12 +102,12 @@ What would you like to do? 5
         - `get_bonus()`
             - `bonus_percentage` * `salary`
         - Where the `Manager` class inherits from the `Employee` class. 
-
-2. Define a list of `Employee` objects for normal employees, and a list of `Manager` objects for managers.
+<!-- wouldn't be better if we add other steps for get_employee, get_manager, add_employee, add_manager functions? -->
```

Comment:
I want them to try and solve it their way, and then we show them how adding in functions makes it more readable, composable, and easier to reason with. What do you think?

Reply:
Totally agree, you're right.